OracleManagementAgent/dockerfiles/latest/Dockerfile

```diff
@@ -45,15 +48,16 @@ ENV LANG en_US.UTF-8
 ENV LC_ALL C
 
 # Copy container scripts to initialize the environment and start Management Agent
-COPY ./container-scripts/*.sh /opt/oracle-mgmtagent-bootstrap/scripts/
-RUN chmod 0544 /opt/oracle-mgmtagent-bootstrap/scripts/*.sh
+COPY ./container-scripts/*.sh /opt/oracle-mgmtagent-staging/scripts/
 
 # Copy user provided scripts
-COPY ./user-scripts/init-agent.sh /opt/oracle-mgmtagent-bootstrap/scripts/
-RUN chmod 0544 /opt/oracle-mgmtagent-bootstrap/scripts/init-agent.sh
+COPY ./user-scripts/init-agent.sh /opt/oracle-mgmtagent-staging/scripts/
 
 # Copy Management Agent Installer Zip bundle
-COPY ./oracle.mgmt_agent.zip /opt/oracle-mgmtagent-bootstrap/packages/
+COPY ./oracle.mgmt_agent.zip /opt/oracle-mgmtagent-staging/packages/
```

Comment:
You should create a `rootfs` folder with the hierarchy so this can be reduced to a single `COPY` directive.

Reply:
The hierarchy we have in place today is for user's readability /container-scripts (do not touch) and /user-scripts (add user directives to it) and when building the container we copy it to the respective locations. Also oracle.mgmt_agent.zip is something user downloads from the Cloud service so we copy it separately. Creating a rootfs with hierarchy would require user to know and understand it and operate around it. Instead this is a lot simpler.